nebula3/data/ResultSet.py

```diff
@@ -192,6 +192,188 @@ def rows(self):
             return []
         return self._data_set_wrapper.get_rows()
 
+    def dict_for_vis(self):
+        """Convert result set to a dictionary format suitable for visualization.
+
+        Example:
+        {
+            'nodes': [
+                {
+                    'id': 'player100',
+                    'labels': ['player'],
+                    'props': {
+                        'name': 'Tim Duncan',
+                        'age': '42',
+                        'id': 'player100'
+                    }
+                },
+                {
+                    'id': 'player101',
+                    'labels': ['player'],
+                    'props': {
+                        'age': '36',
+                        'name': 'Tony Parker',
+                        'id': 'player101'
+                    }
+                }
+            ],
+            'edges': [
+                {
+                    'src': 'player100',
+                    'dst': 'player101',
+                    'name': 'follow',
+                    'props': {
+                        'degree': '95'
+                    }
+                }
+            ],
+            'nodes_dict': {
+                'player100': {
+                    'id': 'player100',
+                    'labels': ['player'],
+                    'props': {
+                        'name': 'Tim Duncan',
+                        'age': '42',
+                        'id': 'player100'
+                    }
+                },
+                'player101': {
+                    'id': 'player101',
+                    'labels': ['player'],
+                    'props': {
+                        'age': '36',
+                        'name': 'Tony Parker',
+                        'id': 'player101'
+                    }
+                }
+            },
+            'edges_dict': {
+                "('player100', 'player101', 0, 'follow')": {
+                    'src': 'player100',
+                    'dst': 'player101',
+                    'name': 'follow',
+                    'props': {
+                        'degree': '95'
+                    }
+                }
+            },
+            'nodes_count': 2,
+            'edges_count': 1
+        }
+
+        :return: dict with keys:
+            nodes, edges, nodes_dict, edges_dict, nodes_count, edges_count
+        """
+
+        def add_to_nodes_or_edges(nodes_dict, edges_dict, item):
```

Comment:
if the resultSet contains `nodes`,`edges`,`paths`,`property`, the dict_for_vis only show the nodes and edges info? how to show property?

Reply:
It's equivalent to our studio/explorer's visualization, those non node/path/edge/list of node path edge data will be ignored.